src/drivers/amd/renoir/acp_sp_dma.c

```diff
@@ -227,18 +224,19 @@ static int acp_dai_sp_dma_set_config(struct dma_chan_data *channel,
 		/* Transmit RINGBUFFER Address and size*/
 		sp_buff_addr = config->elem_array.elems[0].src & ACP_DRAM_ADDRESS_MASK;
 		io_reg_write((PU_REGISTER_BASE + ACP_I2S_TX_RINGBUFADDR), sp_buff_addr);
-		io_reg_write((PU_REGISTER_BASE + ACP_I2S_TX_RINGBUFSIZE), sp_buff_size);
+		io_reg_write((PU_REGISTER_BASE + ACP_I2S_TX_RINGBUFSIZE), sp_buff_size_playback);
```

Comment:
I don't see the point of this change, the code looks at the direction so it's not like you can have different values for playback and capture simulaneously.

Reply:
capture and playback can have different period sizes.